devnotes/mvpd_generation.md

```diff
@@ -0,0 +1,20 @@
+# MVPD data modification
```

Comment:
A section with conclusions should be the most important one in a report (at the end of the report). A first section should describe what goals do we have with those tests - why are we doing that and what we want to achieve.

Reply:
Added a section with a conclusion